ql_demos/cpp/.project

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>cpp</name>
```

Comment:
I thought the answer to that question was that it is enough to change this line, but it seems to depend on the Eclipse version that one's using... That would be the Eclipse way to do it, and indeed it seems to work fine in Eclipse Oxygen, but for some reason not in Eclipse 2019-06 :/

Reply:
I just noticed that the project name doesn't match the path name in the ql repo:

https://github.com/Semmle/ql/blob/master/cpp/ql/src/.project

So shall we just try it without renaming the directory for now? Do you think this could be a temporary bug in the latest version of Eclipse?

Reply:
By the way, I don't mind renaming the directories if that makes this easier.

Reply:
I think it totally looks like a bug in that version yes. Let's try renaming that way for now - I'll push a commit.